api/internal/localizer/localizer_test.go

```diff
@@ -24,8 +28,22 @@ spec:
   - name: nginx
     image: nginx:1.14.2
     ports:
-    - containerPort: 80
-`
+    - containerPort: 80`
+	replacement = `source:
```

Comment:
nit: add a newline between consts for readability

Reply:
Done.